action.yml

```diff
@@ -234,6 +239,21 @@ runs:
           fi
         fi
 
+    - if: ${{ inputs.cache == 'true' }}
+      id: gen-cache-key
+      shell: python {0}
```

Comment:
I don't think you need the `{0}` here. [According to the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iddefaultsrunshell), The accepted value `python` defers to `python {0}` by default.